openbb_platform/platform/provider/openbb_provider/standard_models/major_indices_historical.py

```diff
@@ -38,15 +38,15 @@ class MajorIndicesHistoricalData(Data):
     """Major Indices end of day price data."""
 
     date: datetime = Field(description=DATA_DESCRIPTIONS.get("date", ""))
-    open: PositiveFloat = Field(description=DATA_DESCRIPTIONS.get("open", ""))
+    open: NonNegativeFloat = Field(description=DATA_DESCRIPTIONS.get("open", ""))
```

Comment:
The same principle applies to "high", "low", "close" values.

Reply:
Done.